assets/helpers/abtest.js

```diff
@@ -125,8 +128,18 @@ function userInTest(audiences: Audiences, mvtId: number, country: IsoCountry) {
   return (mvtId > testMin) && (mvtId < testMax);
 }
 
+function randomNumber(seed: number, rounds: number): number {
```

Comment:
Why rounds rather than independence for the parameter name?

Reply:
Probably independence makes more sense.  I was originally thinking about the problem in the context of the hashing algorithms where the term 'rounds' is used for a loosely equivalent purpose.